profiles/karthik-315.json

```diff
@@ -0,0 +1,7 @@
+{
+  "image": "https://user-images.githubusercontent.com/23008566/178808891-ebb58488-680e-4906-bcde-bc00dc9663b5.png",
+  "issueId": 1231,
+  "name": "Karthik S Kumar",
+  "username": "Karthik-315"
+}
+
```

Comment:
There's an extra line here. GitHub doesn't display the endline.
```suggestion
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
Ran prettier and sent another PR. Kindly review again.